src/main/java/org/candlepin/insights/ApplicationConfiguration.java

```diff
@@ -56,6 +56,13 @@ public void onStartup(ServletContext servletContext) throws ServletException {
 
     @Bean
     public ApiClientFactory hostInventoryClientFactory() {
-        return new ApiClientFactory(env.getProperty("hostInventoryServiceUrl"));
+        String hostName = env.getProperty("hostInventoryServiceHostname", "localhost");
+        String port = env.getProperty("hostInventoryServicePort", "8080");
+        String baseUrl = env.getProperty("hostInventoryServiceBaseUrl",
+            "/r/insights/platform/inventory/api/v1");
+        boolean secure = env.getProperty("hostInventoryServiceSecure", Boolean.class, false);
+
+        String protocol = secure ? "https" : "http";
+        return new ApiClientFactory(String.format("%s://%s:%s%s", protocol, hostName, port, baseUrl));
```

Comment:
These changes are unneeded per previous comments.

Reply:
REVERTED - Will be included in next commit.